chat-app/ChatApp/Managers/Network/Entities/UpdateUserEntity.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  UpdateUserEntity.swift
+//  ChatApp
+//
+//  Created by Ramon Jr Bahio on 8/20/24.
+//
+
+import KeychainAccess
+import Foundation
+
+class UpdateUserEntity: RequestableApiEntity {
+    typealias ResponseEntity = UpdateUserRespondableEntity
+
+    static var method: BaseNetworkOperation.Method { AppConstant.shared.forRegister ? .post : .put }
```

Comment:
For register value should be a constructor parameter. This would help devs understand that this value affects the network call.